wappsto/connection/message_data.py

```diff
@@ -61,7 +62,8 @@ def __init__(
             text: The message to send. (default: {None})
             parent: The parent object. (default: {None})
             trace_id: The trace ID used to debug. (default: {None})
-            control_value_id: The control data ID (default: {None})
+            control_value_id: The control data ID. (default: {None})
+            get: Boolean indication if verb should be GET. (default: {False})
```

Comment:
What is get used for?

Reply:
it is used for deciding if request in the sending queue is used for getting data (maybe it could be changed to checking if data was provided or allowing to provide the specific verb to use, for example- "GET", "POST", etc. )

Reply:
But how do you know if it is a put or post?

Reply:
previously it was hardcoded in the method and often post could not be sent since there was no method doing it, but I could make enum storing "GET", "PUT", "POST", "DELETE" instead of having this get parameter

Reply:
Yes I think that we need that it is an enum instead.